index.js

```diff
@@ -96,4 +96,35 @@ function hide(toHide) {
   // Remove show class and add hide class
   toHide.classList.remove('show');
   toHide.classList.add('hide');
-}
\ No newline at end of file
+}
+// dynamic content
+projects.forEach((card, index) => {
+  const workCard = addElement('div', ['work-2'], workContainer);
+  const workContent = addElement('div', ['work2-content'], workCard);
+  const workHeading1 = addElement('h1', [], workContent);
+  workHeading1.textContent = card.name;
+  const workHeading2 = addElement('h2', [], workContent);
+  workHeading2.textContent = card.name;
+  const workParagraph1 = addElement('p', [], workContent);
+  workParagraph1.textContent = card.discription;
+  const listUrl = addElement('ul', [], workContent);
+  const listItem1 = addElement('li', ['list2'], listUrl);
+  listItem1.textContent = 'html';
+  const listItem2 = addElement('li', ['list2'], listUrl);
+  listItem2.textContent = 'bootstrap';
+  const listItem3 = addElement('li', ['list2'], listUrl);
+  listItem3.textContent = 'ruby on rails';
+  const buttonWork = addElement('div', ['work2-button'], workCard);
+  const buttonWork1 = addElement('button', [], buttonWork);
+  buttonWork1.setAttribute('type', 'button');
+  buttonWork1.textContent = 'See Project';
+
+  buttonWork1.onclick = () => showWorkPopup(index);
+});
```

Comment:
- Great job creating your cards dynamically from JavaScript. There are still a few issues with the styling of your cards and popup 👍 
- [ ] (OPTIONAL) In the mobile version your popup just covers the top part of your screen but in the Figma template it is supposed to be positioned in the middle just like you did in the desktop popup. It would be great if you could fix that 💯 
- Your design:
![Annotation 2022-04-15 054543](https://user-images.githubusercontent.com/85113567/163519982-30d6dd97-bdba-4cf7-95f1-c74964272b36.jpg)
- Figma design:
![Annotation 2022-04-15 054556](https://user-images.githubusercontent.com/85113567/163519995-09c70d0e-300e-4242-829d-d193042318d1.jpg)

- [ ] (OPTIONAL) Also kindly implement the functionality of the first project button. Right now it does not open any popup when I click on it. It would be great too if you apply the correct button colors and their correct interactions 😄 
- First project button:
![Annotation 2022-04-15 054627](https://user-images.githubusercontent.com/85113567/163520125-e07adb84-c251-42c0-97dc-837b61b338a5.jpg)
- Button interactions:
![Annotation 2022-04-15 054950](https://user-images.githubusercontent.com/85113567/163520155-6dfe21d8-0885-464c-9c8d-067a8dd40c57.jpg)

Reply:
@Kingstalux  thanks I will work on these!
Really appreciate for checking my work thoroughly!